Updates to Deployment types table

saikondapalli11 opened the pull request:
Ran few updates on Deploymen types table

saikondapalli11 committed:
Updates to Deployment types table

Ran few updates on Deploymen types table

rishky1001 commented on README.md:
```diff
@@ -22,10 +22,10 @@ Below are a list of pre-defined/restricted deployment options based on typical d
-| Small to Mid-Size | Supporting up to 1000 concurrent users.  This deployment will use NFS (no high availability) and MySQL (8 vCores), without other options like elastic search or redis cache.|[![Deploy to Azure Minimally](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Frishky-msft%2FMoodle%2Fusertest%2F ...
-|Large size deployment (with high availability)| Supporting more than 2000 concurrent users. This deployment will use Gluster (for high availability, requiring 2 VMs), MySQL (16 vCores) and redis cache, without other options like elastic search. |[![Deploy to Azure Minimally](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.github ...
-| Maximum |This maximal deployment will use Gluster (for high availability, adding 2 VMs for a Gluster cluster), MySQL with highest SKU, redis cache, elastic search (3 VMs), and pretty large storage sizes (both data disks and DB).|[![Deploy to Azure Maximally](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com% ...
+| Minimal  | This deployment will use NFS, Azure Database for MySQL Flexible Server, and smaller autoscale web frontend VM sku (1 core) that'll give faster deployment time (less than 30 minutes) and requires only 2 VM cores currently that'll fit even in a free trial Azure subscription.|[![Deploy to Azure Minimally](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microso ...
```
Do we want to mention the sku type along with vCores.
For example in maximal we are using memory optimized and in other we are using General purpose?

saikondapalli11 replied:
I have added that information as well, review once you are online.

rishky1001 commented on README.md:
```diff
@@ -22,10 +22,10 @@ Below are a list of pre-defined/restricted deployment options based on typical d
+|Large size deployment (with high availability)| Supporting more than 2000 concurrent users. This deployment will use Gluster (for high availability, requiring 2 VMs), Azure Database for MySQL Flexible Server (16 vCores) and redis cache, without other options like elastic search. |[![Deploy to Azure Minimally](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Te ...
+| Maximum |This maximal deployment will use Gluster (for high availability, adding 2 VMs for a Gluster cluster), Azure Database for MySQL (64 vCores), redis cache, elastic search (3 VMs), and pretty large storage sizes (both data disks and DB).|[![Deploy to Azure Maximally](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubus ...
 
 NOTE: Depending on the region you choose to deploy the stack in - the deployment might fail due to SKUs being hardcoded in the template where they are not available. For example, today our small-mid-size deployment option hard codes Gen-4 Azure MySQL SKUs into the template, and if a region where that is currently not available in (i.e. westus2) is used, your deployment will fail.  If your deploym ...
```
Are these reference for Gen-4 for MySQL is still valid for Flexible server?

saikondapalli11 replied:
Not really, I will remove this statement, because we support v4 in every region and v5 where possible, at the moment I see all our predefined templates using v4

rishky1001 commented on README.md:
```diff
@@ -22,10 +22,10 @@ Below are a list of pre-defined/restricted deployment options based on typical d
+| Maximum |This maximal deployment will use Gluster (for high availability, adding 2 VMs for a Gluster cluster), Azure Database for MySQL (64 vCores), redis cache, elastic search (3 VMs), and pretty large storage sizes (both data disks and DB).|[![Deploy to Azure Maximally](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubus ...
 
 NOTE: Depending on the region you choose to deploy the stack in - the deployment might fail due to SKUs being hardcoded in the template where they are not available. For example, today our small-mid-size deployment option hard codes Gen-4 Azure MySQL SKUs into the template, and if a region where that is currently not available in (i.e. westus2) is used, your deployment will fail.  If your deploym ...
 
```
There are few other reference in this readme for MySQL single server. Can we replace it with Flexible Server.

For example :  Line no 45

saikondapalli11 replied:
Line number 45 points to the main landing page of Azure Database for MySQL, it talks about the service but does not define whether it's single or flexible server, so we can leave that line as it is now.

saikondapalli11 committed:
Update v2.0 from Sai

1. Updated pre-defined template to specify Azure Database for MySQL Flexible Server along with SKU Tier and the cores corresponding to that deployment. 
2. Updated NOTE under the pre-defined template section
3. Added NOTE to the fully configurable template section so a reader can review the options prior to leveraging the template
4. Updated Database part under "Observations" section
5. Updated common questions section: question 11 updated, added a new question 12 to talk about Flexible Server deployment in VNet

rishky1001 reviewed (approved):
LGTM
